### ISSUE TITLE
Assertion failed with INT2 and INT8 typekinds

### ISSUE THREAD

**Issue by sercankd** [larshp/ABAP-Swagger#69]
Is this an intentional exclusion that they are not mapped in MAP_ELEMENT method of ZCL_SWAG_MAP_TYPE?

**Comment by larshp** [comment]
no, just think its missing, guess they should be mapped to integer, note that int8 does not exist in older versions, so hardcode the constant instead of referencing the class constant

pull request welcome

https://github.com/larshp/ABAP-Swagger/blob/39d6cce7f91b91a652da4190f14c3e9583f83173/src/zcl_swag_map_type.clas.abap#L93
